lib/matplotlib/contour.py

```diff
@@ -57,36 +57,37 @@ def clabel(self, *args, **kwargs):
 
         Call signature::
 
-          clabel(cs, **kwargs)
+          clabel(cs, [levels,] **kwargs)
```

Comment:
Perhaps would make sense to change the implementation to have a signature of
```
clabel(cs, levels=None, **kwargs)
```
?
(seems backcompatible and all...)

Reply:
50/50 on changing the signature, not going to hold this PR over it.

Reply:
That's a bit complicated. The docstring belongs to `ContourLabeler.clabel` which is a Mixin to the `ContourSet`. In turn, `Axes.clabel` calls the method but copies the docstring. While the original method is `(*args, **kwargs)`, Axes has `(CS, *args, **kwargs)` with the same parameters.

I think it's possible to untangle this, but would rather do it in a separate PR, because I have to touch the code.